### ISSUE TITLE
Allow UMAP plotting that cryoDRGN computed

### ISSUE THREAD

**Issue by tomgoddard** [charbj/wiggle#5]
Wiggle offers to computing the UMAP plot of the particle latent space but says it is slow.  How about allow using the UMAP coordinates that are provided by cryoDRGN itself in the umap.pkl file?  UMAP often gives a nicer depiction of conformation clusters than PCA.

**Comment by tomgoddard** [comment]
I agree that using umap.pkl from cryoDRGN does not replace having Wiggle able to compute the umap.  But if cryoDRGN already did it the point is to use it.  The standard cryoDRGN analysis computes it and given that cryoDRGN takes tens of hours to compute the neural net, throwing in a umap calculation is not a big deal.  So I think in many cases umap.pkl is free.  Offering umap calculation in ChimeraX where everything is intended to be fast and interactive is much less useful if it takes minutes or hours.

It is interesting that the RAPIDS umap is much faster but unfortunate that it is so difficult to install.

**Comment by beckernick** [comment]
Hi! I just came across this issue based on the RAPIDS reference.

I wanted to mention that we're actively working on pip packages and they're targeted for our 22.10 release in mid-October. Happy to help answer questions about cuML (which provides accelerated UMAP and many other algorithms) or other RAPIDS frameworks.

**Comment by beckernick** [comment]
As an update:
- cuML 22.10 is now available and includes support for a [much wider set of distance metrics in UMAP](https://docs.rapids.ai/api/cuml/nightly/api.html#umap) (in case that customization is useful to you)
- Experimental [RAPIDS pip packages](https://rapids.ai/pip.html) are now available. You can now `pip install cuml-cu11 --extra-index-url=https://pypi.ngc.nvidia.com`

We're excited about these updates and hope they work well for you!
